codice/ddf-ui
# [3.6.x] DDF-UI-339 Update gazetteer and fix dropdown.js infinite rerendering

Description

- Update gazetteer and keyword to hooks
- Attempt to type gazetteer and keyword
- Fix issue with dropdown.js infinitely rerendering
- Add placeholder and loadingMessage props to gazetteer

Reviewers
@stevenmalmgren @andrewkfiedler @vinamartin @a-asaad 

What are the relevant tickets?
Fixes: #339 

Testing

1. Verify that the map gazetteer still works properly (loads results, selecting one applies and zooms to results)
1. Verify that the search gazetteer still works (selecting anyGeo -> keyword) still works (note: this doesn't use the online gazetteer, so it's expected that there will be no results)
